mathesar_ui/src/components/select/Select.svelte

```diff
@@ -28,22 +28,23 @@
 
   $: setOptions(options);
 </script>
-
-<Dropdown bind:isOpen contentClass="select">
-  <svelte:fragment slot="trigger">
-    {value?.[labelKey]}
+<span id="totalRows">Total Rows
+<Dropdown id="listbox-options"  bind:isOpen contentClass="select">
+  <svelte:fragment slot="trigger" >
+    {value?.[labelKey] }
   </svelte:fragment>
   
   <svelte:fragment slot="content">
-    <ul>
+    <ul id="listbox-options" tabindex="0" role="listbox" aria-labelledby="totalRows" aria-expanded="true">
```

Comment:
`id` is a property which is supposed to be unique across all dom elements. Here, the same id will be used, if we have multiple instances of Select component.

Inorder to handle this, we can generate unique ids when component is instantiated. Here is a reference [select component](https://github.com/pavish/select-madu/blob/master/src/components/Main.svelte), which uses dynamic ids. It generates id on the module context. Here is the svelte documentation on [how module context works](https://svelte.dev/docs#script_context_module).